homeassistant/components/remote_rpi_gpio/__init__.py

```diff
@@ -0,0 +1,122 @@
+"""Support for controlling GPIO pins of a Raspberry Pi."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.const import (
+    CONF_HOST, CONF_BINARY_SENSORS, CONF_SWITCHES)
+from homeassistant.core import HomeAssistant
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.discovery import load_platform
+
+REQUIREMENTS = ['gpiozero==1.4.1', 'pigpio==1.42', 'RPi.GPIO==0.6.5']
```

Comment:
I see only the import of `gpiozero`. We don't pin another library, because that is need be done inside `gpiozero`